apps/builder/app/builder/features/style-panel/controls/position/position.tsx

```diff
@@ -0,0 +1,115 @@
+import {
+  TupleValue,
+  type StyleValue,
+  TupleValueItem,
+} from "@webstudio-is/css-data";
+import { Flex, Label, PositionGrid, theme } from "@webstudio-is/design-system";
+import type { ControlProps } from "../../style-sections";
+import { styleConfigByName } from "../../shared/configs";
+import { getStyleSource } from "../../shared/style-info";
+import { CssValueInputContainer } from "../../shared/css-value-input";
+import type { SetValue } from "../../shared/use-style-data";
+
+const toPosition = (value: TupleValue) => {
+  return {
+    left: value.value[0].value,
+    top: value.value[1].value,
+  };
+};
+
+// @todo SetValue has legacy string value support, that needs to be removed
+const toTuple = (
```

Comment:
The comment is confusing,  `string` now can come because of computed style  not because of setValue
(setProperty uses toStyleValue inside, so no strings in our styles)

Reply:
we also have cases where we pass a string from control, either way the important thing here to not pass a string ever again, no matter why